docs: update sessions.md (#77626)
airbytehq/airbyte: docs/ai-agents/admin/sessions.md

- **Status**: Notes Active or Deleted. Active chats are resumable.
- **Tools**: The connectors the agent used during the session. Hover over a connector icon to see its name.
- **Messages**: The number of messages exchanged in the session. Automation sessions typically have fewer messages than chat sessions, since they don't involve a back-and-forth with a user.
- **Tool Calls**: The number of tool calls the agent made during the session. Drill into [tool calls](./tool-calls.md) to see individual calls.
- **Input Tokens**: The total tokens the agent received as input, across every turn in the session.
- **Output Tokens**: The total tokens the agent produced, across every turn in the session.
- **Date**: When the session last updated. Sessions that are still running show the most recent activity time. Dates display in your browser's local [time zone](../concepts/time-zones).
- **Actions**: Per-row buttons to review or resume the session.
    - Click the **View** icon to open the session and see its messages and tool calls.
    - Click the **Chat** icon to jump to the chat or automation the session belongs to.

> Use the **Type**, **Status**, and **Tools** filters above the table to narrow the list. The Type filter scopes to chats or automations. The Status filter scopes to Active, which may be resumed, or Deleted. The Tools filter scopes to sessions that used one or more specific connectors.

Reviewer: ```suggestion
Use the **Type**, **Status**, and **Tools** filters above the table to narrow the list. The Type filter scopes to chats or automations. The Status filter scopes to Active, which you can resume, or Deleted. The Tools filter scopes to sessions that used one or more specific connectors.
```

## Access previous sessions

> To open the Sessions page, click **Settings** then **Sessions** in the left navigation of your workspace. The page lists sessions across every automation and chat in your workspace. Sessions never expire, so you can review work from any point in your workspace's history.

Reviewer: ```suggestion
To open the Sessions page, click **Settings** > **Sessions** in the left navigation of your workspace. The page lists sessions across every automation and chat in your workspace. Sessions never expire, so you can review work from any point in your workspace's history.
```
